xsd-parser-rs/src/tests/type_name_clash/mod.rs

```diff
@@ -16,8 +17,10 @@ fn deserialization_works() {
     assert_eq!(
         de,
         expected::FooType {
-            a: Some("aaa".to_string()),
-            b: Some("bbb".to_string())
+            bar: expected::foo_type::BarType {
+                b: None,
+                field: None
+            }
```

Comment:
I think this is wrong to expect `None`s here because the data should be deserialized from `example.xml`.

Reply:
Try regenerating `example.xml` with
```
xsd2inst input.xsd -name Foo > example.xml
```